airbyte-cdk/java/airbyte-cdk/s3-destinations/src/testFixtures/kotlin/io/airbyte/cdk/integrations/destination/s3/S3BaseJsonlDestinationAcceptanceTest.kt

```diff
@@ -43,7 +42,7 @@ abstract class S3BaseJsonlDestinationAcceptanceTest protected constructor() :
             getReader(`object`).use { reader ->
                 var line: String?
                 while ((reader.readLine().also { line = it }) != null) {
-                    jsonRecords.add(Jsons.deserialize(line)[JavaBaseConstants.COLUMN_NAME_DATA])
+                    jsonRecords.add(Jsons.deserialize(line))
```

Comment:
checking my understanding: this is just so that our tests can assert against the `_airbyte*` fields, not because of any change to the record structure?

Reply:
Correct. All of these implementations formerly returned only the client data.